iroh-ctl/src/main.rs

```diff
@@ -67,6 +93,10 @@ async fn main() -> anyhow::Result<()> {
     )
     .unwrap();
 
+    let metrics_handle = iroh_metrics::MetricsHandle::new(MetricsConfig::default())
```

Comment:
we really don't have any `ctl` side metrics and passing anything from resolver etc might just introduce noise into the existing metrics. Can we drop this / avoid pushing metrics for now and I can eventually do a separate pass if we think this is a useful thing.

Reply:
I would really like to have this integrated the same way, especially for local testing, when it comes to metrics. 
Why do you think this adds noise? These are real requests, just created differently

Reply:
Well it would be interesting to have eg bitswap metrics from both ends.

The noise is mostly that I need to filter out the ctl stuff from the regular on the grafana dashboards. Also need to audit code paths making sure ctl is updating relevant metrics.

Reply:
I added this partially to get logging on `iroh-ctl`. I can always just import a env logger and use that, but it felt better to be consistent.

Should I keep the metrics handler, or adjust to a different logger?

Reply:
Oh ok, so to get logging working you don't need to set up the full pipe. Should be enough to call init metrics without spawning the handler. That way you will get our default logger, without pushing metrics for now. Im on mobile so a bit hard to highlight the specific line. I'll follow up tonight.